obj/objects.h

```diff
@@ -68,9 +69,9 @@ namespace objects
     class EnergyCollector : public Sphere
     {
     public:
-        EnergyCollector(const core::Vec3 &origin, double energy) : _id(id), _energy(energy)
+        EnergyCollector(const core::Vec3 &origin, double energy = 0) : _energy(energy) // TODO: delete argument for energy and assing 0 to energy by default
```

Comment:
Do not put TODO, just do it before merging this branch into the master.

Reply:
ok hahah

Reply:
done.